frontend/src/components/SearchCard.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Drawer from '@material-ui/core/Drawer';
+import { makeStyles, useTheme, Theme, createStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+
+const drawerWidth = 350;
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        root: {
+            display: 'flex',
+        },
+        appBar: {
+            transition: theme.transitions.create(['margin', 'width'], {
+                easing: theme.transitions.easing.sharp,
+                duration: theme.transitions.duration.leavingScreen,
+            }),
+        },
+        appBarShift: {
+            width: `calc(100% - ${drawerWidth}px)`,
+            marginLeft: drawerWidth,
+            transition: theme.transitions.create(['margin', 'width'], {
+                easing: theme.transitions.easing.easeOut,
+                duration: theme.transitions.duration.enteringScreen,
+            }),
+        },
+        menuButton: {
+            marginRight: theme.spacing(2),
+        },
+        hide: {
+            display: 'none',
+        },
+        drawer: {
+            width: drawerWidth,
+            flexShrink: 0,
+        },
+        drawerPaper: {
+            width: drawerWidth,
+        },
+        drawerHeader: {
+            display: 'flex',
+            alignItems: 'center',
+            padding: theme.spacing(0, 1),
+            // necessary for content to be below app bar
+            ...theme.mixins.toolbar,
+            justifyContent: 'flex-end',
+        },
+        title: {
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'flex-start',
+            justifyContent: 'space-between',
+            marginBottom: 15,
+            marginLeft: 15,
+        },
+        miniTitle: {
+            margin: 0,
+        }
+        
+    })
+)
+
+const SearchCard = () => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const classes = useStyles();
+    const theme = useTheme();
+    const [murals, setMurals] = useState<any>([]);
+    const [open, setOpen] = React.useState(false);
+    // const [tours, setTours] = useState(null);
+    // const [collections, getCollections] = useState(null);
+    useEffect(() => {
```

Comment:
Will this fire at every state change?

Reply:
yes correct! but I thought it is necessary because if we were to add new data to the database, it will keep updating.

Reply:
I think you can add an empty array as a second argument to only fire once like `useEffect(() => {}, [])`. I'm getting an infinite loop in the browser lol

Reply:
As for fetching fresh data from the DB in real-time, this would be super cool! Maybe we can look at reconfiguring the backend to send out notifications to do this at the end of the semester.